tests/benchdnn/inputs/graph/complex_fusion/generated/sdpa-scl.json

```diff
@@ -0,0 +1 @@
+{"version": "3.2.0", "engine_kind": "gpu", "fpmath_mode": "strict", "fpmath_mode_apply_to_int": "false", "input_ports": [0, 3, 4, 6], "output_ports": [50], "graph": [{"id": 200, "name": "matmul_qk", "kind": "MatMul", "attrs": {"transpose_a": {"type": "bool", "value": 0}, "transpose_b": {"type": "bool", "value": 0}}, "inputs": [{"id": 3, "dtype": "f16", "shape": [1, 16, 384, 64], "stride": [393216, 24576, 64, 1], "layout_type": "strided", "property_type": "variable"}, {"id": 0, "dtype": "f16", "shape": [1, 16, 64, 384], "stride": [393216, 24576, 384, 1], "layout_type": "strided", "property_type": "variable"}], "outputs": [{"id": 15, "dtype": "f32", "shape": [1, 16, 384, 384], "stride": [2359296, 147456, 384, 1], "layout_type": "strided", "property_type": "variable"}]}, {"id": 300, "name": "scale_s", "kind": "Divide", "attrs": {"auto_broadcast": {"type": "string", "value": "numpy"}}, "inputs": [{"id": 15, "dtype": "f32", "shape": [1, 16, 384, 384], "stride": [2359296, 147456, 384, 1], "layout_type": "strided", "property_type": "variable"}, {"id": 4, "dtype": "f16", "shape": [1], "stride": [1], "layout_type": "strided", "property_type": "undef"}], "outputs": [{"id": 16, "dtype": "f32", "shape": [1, 16, 384, 384], "stride": [2359296, 147456, 384, 1], "layout_type": "strided", "property_type": "variable"}]}, {"id": 500, "name": "softmax", "kind": "SoftMax", "attrs": {"axis": {"type": "s64", "value": 3}}, "inputs": [{"id": 16, "dtype": "f32", "shape": [1, 16, 384, 384], "stride": [2359296, 147456, 384, 1], "layout_type": "strided", "property_type": "variable"}], "outputs": [{"id": 27, "dtype": "f16", "shape": [1, 16, 384, 384], "stride": [2359296, 147456, 384, 1], "layout_type": "strided", "property_type": "variable"}]}, {"id": 700, "name": "matmul_v", "kind": "MatMul", "attrs": {"transpose_a": {"type": "bool", "value": 0}, "transpose_b": {"type": "bool", "value": 0}}, "inputs": [{"id": 27, "dtype": "f16", "shape": [1, 16, 384, 384], "stride": [2359296, 147456, 384, 1], "layout_type": "strided", "property_type": "variable"}, {"id": 6, "dtype": "f16", "shape": [1, 16, 384, 64], "stride": [393216, 24576, 64, 1], "layout_type": "strided", "property_type": "variable"}], "outputs": [{"id": 50, "dtype": "f16", "shape": [1, 16, 384, 64], "stride": [393216, 24576, 64, 1], "layout_type": "strided", "property_type": "variable"}]}]}
```

Comment:
Same question :), is it intended that softmax doesn't have the inf_as_zero attribute? If not, I think this one should be similar to [sdpa-plain-wo-mask-f16.json](https://github.com/uxlfoundation/oneDNN/blob/main/tests/benchdnn/inputs/graph/complex_fusion/mha/sdpa-plain-wo-mask-f16.json)